src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -104,7 +104,7 @@ public OrganisationResponse createOrganisationFrom(
         return new OrganisationResponse(organisation);
     }
 
-    private Organisation saveOrganisation(Organisation organisation) {
+    public Organisation saveOrganisation(Organisation organisation) {
```

Comment:
These methods were made public as per discussion and agreement with @shashank-rastogi:

> a new instance of Organisation is created in the method and then passed to saveOrganisation method, so I am unable to do a verify on this new Organistion instance as I cannot mock it